dictionaries/google/src/legacy.txt

```diff
@@ -0,0 +1,349 @@
+# This is a list of words that had been in the original dictionary, but are part of other dictionaries.
+# They are included for compatibility reasons.
+# Please do not add new words to this list.
```

Comment:
As people adds new words at the end, maybe you could add a comment at the end of the file.

Reply:
Good idea.